pages/QuizSelection.tsx

```diff
@@ -1,15 +1,32 @@
 import React from 'react';
+import QuizCard from '../components/sections/QuizCard';
+import styles from '../styles/QuizSelection.module.scss';
+import Link from 'next/Link';
```

Comment:
Uppercase 'L' in next/link will break deployment build on vercel
<img width="895" alt="Screen Shot 2021-09-23 at 10 14 26 PM" src="https://user-images.githubusercontent.com/9421706/134611651-f3ec1c1c-1a66-49c8-93cb-70caa87169d0.png">


```suggestion
import Link from 'next/link';
```